### PR TITLE
added: cart page

### PR DESCRIPTION
**Pull request by harshalini**
Added cart page for vivir ecommerce.
it displays the products added to the cart.
You can see the product, quantity and the subtotal.
check it here ->
https://adoring-mestorf-f1eb6b.netlify.app/cart/cart.html
Screenshot (304)

### CODE REVIEW

File: cart/cart.css [hunk 1]
```diff
@@ -0,0 +1,61 @@
+@import url(../products.css);
+
+.cart-container {
+    margin: auto;
+    margin-top: 7rem;
+}
+.cart-product img {
+    width: 5rem;
+    height: 6rem;
+    margin-right: 0.65rem;
```
Comment (fahd23) [inline cart/cart.css]: you can use px here, right?

**Review by fahd23 — APPROVED** [review]
UI and code looks awesome🔥

**Review by munjalSaksham71 — APPROVED** [review]
Looks Cool To me !  You can merge it off

File: cart/cart.html [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+    <div class="container product-navbar">
+        <div class="navbar">
+            <div class="logo">
+             <a href="../home/index.html">vivir</a>
+            </div>
+            <nav>
+                <ul class="nav-pages">
+                    <li><a href="../index.html">Home</a></li>
+                    <li><a href="../products.html">Products</a></li>
+                    <li><a style="color: var(--orange);
```
Comment (t007rushi) [inline cart/cart.html]: avoid using inline styling, for good practices

File: products.html [hunk 3]
```diff
@@ -17,8 +17,9 @@
             <nav>
                 <ul class="nav-pages">
                     <li><a href="index.html">Home</a></li>
-                    <li><a href="#">Products</a></li>
-                    <li><a href="cart.html">Cart</a></li>
+                    <li><a style="color: var(--orange);
```
Comment (t007rushi) [inline products.html]: same as above

File: cart/cart.css [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+}
+.cart-product input {
+    width: 2.5rem;
+    height: 2rem;
+    padding: 5px;
```
Comment (t007rushi) [inline cart/cart.css:15]: yo can use px below 10px then don't use 0. rem values, following uniformity will help

File: cart/cart.css (hunk 1 above)
Comment (harshalini) [reply cart/cart.css]: Yes, will change it now

**Review by t007rushi — CHANGES_REQUESTED** [review]

File: cart/cart.html (hunk 2 above)
Comment (harshalini) [reply cart/cart.html]: Thanks for suggesting, made the changes, please approve this PR

**Review by t007rushi — APPROVED** [review]

**Pull request merged**